worker/src/test/java/gov/cms/ab2d/worker/properties/PropertiesChangeDetectionTest.java

```diff
@@ -0,0 +1,58 @@
+package gov.cms.ab2d.worker.properties;
+
+import gov.cms.ab2d.common.model.Properties;
+import gov.cms.ab2d.common.repository.PropertiesRepository;
+import gov.cms.ab2d.common.util.AB2DPostgresqlContainer;
+import org.junit.Assert;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.core.env.ConfigurableEnvironment;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import static gov.cms.ab2d.common.util.Constants.*;
+
+@SpringBootTest(properties = "app.scheduling.enable=false")
+@Testcontainers
+public class PropertiesChangeDetectionTest {
+
+    @Autowired
+    private PropertiesChangeDetection propertiesChangeDetection;
+
+    @Autowired
+    private PropertiesRepository propertiesRepository;
+
+    @Autowired
+    private ConfigurableEnvironment configurableEnvironment;
+
+    @Container
+    private static final PostgreSQLContainer postgreSQLContainer = new AB2DPostgresqlContainer();
+
+    @Test
+    public void testChangeInProperties() {
```

Comment:
This test case also needs to verify the property changes got reflected in the corresponding beans (again, see AB2D-1035)